src/js/_enqueues/admin/user-profile.js

```diff
@@ -332,6 +340,75 @@
 		}
 	}
 
+	/**
+	 * Binds Caps Lock detection to a given password input field.
+	 *
+	 * @param {jQuery} $input The password input field.
+	 */
+	function bindCapsLockWarning( $input ) {
+		var $capsWarning = $( '#caps-warning' ),
+			capsLockOn   = false;
+
+		$input.on( 'keydown', function( e ) {
+			var ev = e.originalEvent || e;
```

Comment:
Will `e.originalEvent` ever be empty? I don't believe so. I think this can be simply:

```suggestion
		$input.on( 'keydown', function( jqEvent ) {
			var event = jqEvent.originalEvent;
```